features.json

```diff
@@ -104,7 +104,8 @@
 				"signExtensions": "74",
 				"simd": "91",
 				"tailCall": ["flag", "Requires flag `chrome://flags/#enable-experimental-webassembly-features`"],
-				"threads": "74"
+				"threads": "74",
+				"typeReflection": ["flag", "Requires flag `chrome://flags/#enable-experimental-webassembly-features`"],
```

Comment:
Now that the new version was deployed, a trailing comma here is breaking the site:

```
SyntaxError: Expected double-quoted property name in JSON at position 3936
```

Reply:
Thanks, fixed.